### ISSUE TITLE
No resource found that matches the given name with 2.4.0

### ISSUE THREAD

**Issue by ovy9086** [Adyen/adyen-android#54]
Using the latest Adyen SDK 2.4.0 produces the following errors : 

```
/Users/ovidiu.latcu/.gradle/caches/transforms-1/files-1.1/ui-2.4.0.aar/1f4a69eb5f49e032283cf860e2dcb450/res/layout/activity_open_invoice_details.xml:39:45-67: AAPT: No resource found that matches the given name (at 'layout_alignBottom' with value '@id/editText_ssnLookup').
    

/Users/ovidiu.latcu/.gradle/caches/transforms-1/files-1.1/ui-2.4.0.aar/1f4a69eb5f49e032283cf860e2dcb450/res/layout/activity_open_invoice_details.xml:40:44-66: AAPT: No resource found that matches the given name (at 'layout_alignRight' with value '@id/editText_ssnLookup').
  

/Users/ovidiu.latcu/.gradle/caches/transforms-1/files-1.1/ui-2.4.0.aar/1f4a69eb5f49e032283cf860e2dcb450/res/layout/activity_open_invoice_details.xml:41:42-64: AAPT: No resource found that matches the given name (at 'layout_alignEnd' with value '@id/editText_ssnLookup').
```

**Comment by ovy9086** [comment]
https://github.com/Adyen/adyen-android/pull/55

**Comment by caiofaustino** [comment]
Just out of curiosity, whats your current?

compileSdkVersion
buildToolsVersion
targetSdkVersion
Support library version

**Comment by ovy9086** [comment]
buildToolsVersion = "28.0.2"
    minSdkVersion = 21
    compileSdkVersion = 28
    targetSdkVersion = 28
    supportLibVersion = "28.0.0"
    googlePlayServicesVersion = "15.0.1"

**Comment by caiofaustino** [comment]
Thanks, the fix for the issue will be in the next release.

**Comment by ovy9086** [comment]
thanks. any ETA for this ?

**Comment by caiofaustino** [comment]
Likely next week.

**Comment by ovy9086** [comment]
any updates on this. this is blocking us from making a release for a customer that makes use of some of the new functionalities

**Comment by caiofaustino** [comment]
Just released on 2.4.1
